examples/v1beta1/sdk/tune-train-from-func.ipynb

```diff
@@ -466,20 +546,28 @@
     "\n",
     "# Start TFJob Training.\n",
     "tfjob_name = \"train-mnist\"\n",
-    "tfjob_client = TFJobClient()\n",
+    "tfjob_client = TrainingClient(namespace=namespace)\n",
     "\n",
-    "tfjob_client.create_tfjob_from_func(\n",
+    "#create_tfjob_from_func\n",
+    "tfjob_client.create_job(\n",
     "    name=tfjob_name,\n",
-    "    func=train_mnist_model,\n",
+    "    namespace=namespace,\n",
+    "    job_kind=\"TFJob\",\n",
```

Comment:
You can pass the job_kind to the TrainingClient(), so it will be re-used everywhere: https://github.com/kubeflow/trainer/blob/release-1.9/sdk/python/kubeflow/training/api/training_client.py#L47

Reply:
Did you address this change @yehudit1987 ?